bin/cli.py

```diff
@@ -59,45 +61,23 @@ def train(
 
 
 @main.command()
-def infer():
-    run_path = Path("./path/to/one/of/your/training/runs")
-    label = 6
+def infer(
+    run_path: str = typer.Option(
+        ..., '-p', '--path', help="File path for inference."
+    ),
+    label: int = typer.Option(
+        6, '-l', '--label', help ="Label of image you want to load"
+    )
+):
 
     # select image to run inference for

```

Comment:
you can remove this comment now that it is no longer relevant. Dangling documentation can get very confusing for another user :)